### ISSUE TITLE
Detailed explanation on setup

### ISSUE THREAD

**Issue by bergses** [DeltaXML/vscode-xslt-tokenizer#40]
Can any helpful soul create a detailed description to set this up in VSCode. For someone new to VSCode its very hard to follow the steps in setting up the XSLT Tasks and config files. 

It would be amazing with a step-by-step setup guide. Thanks <3

**Comment by pgfearo** [comment]
I'm working on some extra introductory videos and documentation - but help from others would be appreciated. 

Here's an introduction [YouTube video](https://www.youtube.com/watch?v=fdxfXaJw6SY&t=821s) - this shows how to setup the VS Code extension for Saxon-JS. 

The Microsoft VS Code documentation is pretty good for general topics not specific to XSLT/XPath setup:

https://code.visualstudio.com/docs

**Comment by pgfearo** [comment]
Created new documentation (XSLT/XPath User Guide for Visual Studio Code)[https://deltaxml.github.io/vscode-xslt-xpath/editing-xslt.html] - this has a detailed 'Running XSLT' section.
